### DISCUSSION TITLE
is it possible to check if all parameter names of functions are explicitly written out?

### DISCUSSION THREAD

**Discussion by ChannyClaus** [astral-sh/ruff#8681] [Q&A]
Suppose you have a function that looks like
```
def test(a, b, c):
    return a + b + c
```
While it is perfectly valid to call it like
```
print(test(1, 2, 3))
```
I'm wondering if it is possible to force via `ruff` to make this call to look more like
```
print(test(a=1, b=2, c=3))
```
(and raise an error if the name of the parameter isn't specified). is there a rule for this in `ruff`? i did try to look for it on the rule page for something that would have "keyword" or "parameter" but wasn't able to find anything.

thanks in advance!

**Comment by charliermarsh** [comment]
It's not currently supported unfortunately -- but it has been requested here: https://github.com/astral-sh/ruff/issues/8382 :)
